cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1074,6 +1074,26 @@
       <description>Security realm used for authentication</description>
   </property>
 
+  <!--
+      Security configurations for Kerberos authentication
+  -->
+  <property>
+    <name>cdap.master.kerberos.keytab</name>
+    <value></value>
+    <description>The full path to the Kerberos keytab file containing the CDAP Master's
+    credentials.</description>
+  </property>
+
+  <property>
+    <name>cdap.master.kerberos.principal</name>
+    <value></value>
+    <description>Ex. "cdap/_HOST@EXAMPLE.com".  The Kerberos principal name that should be used to
+    login the CDAP Master process.  The principal name should be in the form
+    "user/hostname@REALM".  If the hostname portion contains the string "_HOST", it will be
+    substituted with the local hostname.</description>
```

Comment:
+1 on moving ``cdap/_HOST@EXAMPLE.com`` to the default value. I would encourage different line breaks to make it easier to read:
````
<description>
Example: "cdap/_HOST@EXAMPLE.com"
The Kerberos principal name that should be used to login the CDAP Master process. 
The principal name should be in the form "user/hostname@REALM".
If the hostname portion contains the string "_HOST", it will be
substituted with the local hostname.
</description>
````